src/ipvs/ip_vs_conn.c

```diff
@@ -759,9 +762,7 @@ struct dp_vs_conn *dp_vs_conn_new(struct rte_mbuf *mbuf,
     rte_atomic32_set(&new->refcnt, 1);
     new->flags  = flags;
     new->state  = 0;
-#ifdef CONFIG_DPVS_IPVS_STATS_DEBUG
-    new->ctime = rte_rdtsc();
-#endif
+    new->ctime = rte_rdtsc_precise();
```

Comment:
I found the time cost of "rte_rdtsc" is a bit high when I did performance test using perf.  Do we really need to know exactly when the connection created?

Reply:
Yes, agree. I find this command is not very useful for debugging. I think we can keep this out first.